scss/_patterns_navigation.scss

```diff
@@ -100,31 +127,37 @@
       }
 
       .p-navigation__link {
```

Comment:
I am aware of the padding issue. I need to change the mark up for that so will be a bigger task.